api/src/repositories/observation-repository.ts

```diff
@@ -141,7 +141,12 @@ export class ObservationRepository extends BaseRepository {
         longitude = EXCLUDED.longitude
     `);
 
-    sqlStatement.append(`RETURNING *;`);
+    sqlStatement.append(`
+      RETURNING
+        *,
+        latitude::double precision,
```

Comment:
working on the add/ edit for observations, I put a parse function in the `db.ts` that looks for the numeric type from postgres and converts it using `parseFloat`. It also means that whenever we need to add the lat/ long from this table we don't need to explicitly convert them, it is handled automatically.

Reply:
For now, let's leave this conversion in; Currently, updating the table is broken in dev. Assuming this PR will get merged before yours, we can remove this conversion in the subsequent PR.